.changeset/mean-panthers-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder': minor
```

Comment:
This should be a patch release until we hit version 1.x 👍

Reply:
Thanks, done!

Reply:
Thanks, done